Description: Upstream changes introduced in version f3f17ec-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 node2dm (f3f17ec-1) stable; urgency=low
 .
   * Initial release.
 .
 The person named in the Author field signed this changelog entry.
Author: Steffen Zieger <me@saz.sh>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- node2dm-f3f17ec.orig/node2dm.js
+++ node2dm-f3f17ec/node2dm.js
@@ -1,3 +1,5 @@
+#!/usr/bin/env node
+
 var dgram = require('dgram')
   , util = require('util')
   , http = require('http')
@@ -373,4 +375,5 @@ fs.stat('quota.lock', function(err, stat
 
     var connection = new C2DMConnection(config);
     var receiver = new C2DMReceiver(config, connection);
-});
\ No newline at end of file
+});
+
